query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -87,17 +70,13 @@ public String resolveProperty(String str) {
     public ReportImpl createManagedService(Map<String, Object> serviceProps, String factoryPid) {
         ReportImpl report = new ReportImpl();
 
-        if (configuratorFactory != null && managedServiceActions != null) {
-            Configurator configurator = configuratorFactory.getConfigurator();
-            configurator.add(managedServiceActions.create(factoryPid, serviceProps));
+        Configurator configurator = configuratorSuite.getConfiguratorFactory()
+                .getConfigurator();
+        configurator.add(configuratorSuite.getManagedServiceActions()
+                .create(factoryPid, serviceProps));
 
-            if (configurator.commit("Service saved with details [{}]", serviceProps.toString())
-                    .containsFailedResults()) {
-                report.addResultMessage(failedPersistError());
-            }
-        } else {
-            LOGGER.debug(
-                    "Unable to create managed service due to missing configuratorFactory or managedServiceActions.");
+        if (configurator.commit("Service saved with details [{}]", serviceProps.toString())
```

Comment:
Nope. You're right. They need to be blanked or they end up in the security log. I'll note them.